### PR TITLE
Handle all supported browsers on Windows (#24)

### LINKED ISSUE #24: Handle all supported browsers on Windows
Currently [we only listen for Chrome processes](https://github.com/sindresorhus/kill-tabs/blob/main/windows.js#L8) on Windows. We should also add support for the other browsers as well.

### PR DESCRIPTION
**Pull request by GeorgeGkas**
Fixes #24

### CODE REVIEW

File: windows.js [hunk 1]
```diff
@@ -4,8 +4,26 @@ import execall from 'execall';
...
+
+	if (options.brave) {
+		browsers.push('Caption=\'brave.exe\'')
+	}
+
+	if (options.edge) {
+		browsers.push('Caption=\'msedge.exe\'')
+	}
+
+	const command = `wmic process where "${browsers.join(' or ')}" get CommandLine,ProcessId /format:list`;
```
Comment (sindresorhus) [inline windows.js]: It would be great if you could switch to `childProcess.execFile` while you're at it. That would simplify the quoting here.

Comment (GeorgeGkas) [reply windows.js]: Something like the following?? (It is a working example)

```js
const args = ['process', 'where', browsers.join(' or '), 'get','CommandLine,ProcessId', '/format:list'];
const stdout = await promisify(childProcess.execFile)('wmic', args, {maxBuffer: TEN_MEBIBYTE});
```

Comment (sindresorhus) [reply windows.js]: Yes

Comment (GeorgeGkas) [reply windows.js]: updated

**Comment by sindresorhus** [comment]
Make sure you manually test that it still works on Windows after these changes.

**Comment by GeorgeGkas** [comment]
Added a commit to handle an edge case where the running machine do not have any browser installed, which will make the wmic command to report on stderr with "No Instance(s) Available.". We don't handle this case as an actual exception, so we do not throw.

**Comment by GeorgeGkas** [comment]
Tests passed. If you are ok, we can resolve this PR.

File: windows.js [hunk 2]
```diff
@@ -4,11 +4,29 @@ import execall from 'execall';
...
+
+	if (options.brave) {
+		browsers.push('Caption=\'brave.exe\'');
+	}
+
+	if (options.edge) {
+		browsers.push('Caption=\'msedge.exe\'');
+	}
+
+	const args = ['process', 'where', browsers.join(' or '), 'get', 'CommandLine,ProcessId', '/format:list'];
```
Comment (sindresorhus) [inline windows.js]: You overwrote https://github.com/sindresorhus/kill-tabs/pull/25/commits/8849b97bf03847dfdf924f0111c5089e0cc5d4e5

Comment (GeorgeGkas) [reply windows.js]: oops I'm pushing a new commit to resolve it.
